src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface FruitDao {
+    int getData(String fruit);
+
+    void putInData(String fruit, int quantity);
+
+    void updateData(String fruit, int newQuantity);

```

Comment:
What the difference between these two methods, let's remove one of them and rename it to saveQuantity

Reply:
CRUD

Reply:
You don't need to add all CRUD methods if your app does not require it